src/components/FlawForm.vue

```diff
@@ -87,16 +87,22 @@ watch(() => props.flaw, () => { // Shallow watch so as to avoid reseting on any
   onReset();
 });
 
+const isPublic = computed(() => !initialFlaw.value?.embargoed);
+const showUnembargoingModal = ref(false);
```

Comment:
I think we can use useModal composable. https://github.com/RedHatProductSecurity/osim/blob/main/src/composables/useModal.ts

Reply:
Hey @tboyetteredhat thanks for the review.

The `useModal` composable is used in the `IssueFieldEmbargo` component, do you mean to substitute the property on `showUnembargoingModal` for `useModal` methods?

I think that might require to move the modal composable from the child to the parent?

Reply:
Oh I see. I think we don't need to change it. Looks good to me.